extensions/interactions/EndExploration/directives/end-exploration-interaction.directive.html

```diff
@@ -1,3 +1,11 @@
-<div ng-if="$ctrl.isInEditorPage" style="color: red; font-size: 0.8em; margin-top: 20px;">
+<div ng-if="$ctrl.isInEditorPage" class="oppia-error-message-text">
   <span><[$ctrl.errorMessage]></span>
 </div>
+
+<style>
+  oppia-interactive-end-exploration .oppia-error-message-text {
+    color: red;
+    font-size: 0.8em;
+    margin-top: 20px;
+  }
```

Comment:
I would assume that we use some kind of error text in other places too, do we need to redifine it here?

Reply:
The properties in this class are different from the properties in an already defined class. So, I think it is necessary to redefine here.

Reply:
How different, could you maybe extract the mutual info between the two classes and use that?

Reply:
Only the color is same
```
.oppia-form-error-text {
  color: red;
  font-size: smaller;
}
```